lib/ui/recommendation-row.js

```diff
@@ -30,32 +35,52 @@ RecommendationRow.prototype = {
     recommendation.
     */
     let row = this;
+
+    if (!this.resultType) {
+      return null;
```

Comment:
Rather than return a specific value like `return null`, you can generally just `return;` in JS. The default return value is `undefined`, which is falsy, and that's often enough to get the job done.